bin/migrate-oats-data/submissions/app_submissions.py

```diff
@@ -61,10 +66,12 @@ def process_alcs_app_submissions(conn=None, batch_size=BATCH_UPLOAD_SIZE):
                 try:
                     adj_rows = get_directions_rows(rows, cursor)
                     direction_data = create_direction_dict(adj_rows)
+                    subdiv_rows = get_subdiv_rows(rows, cursor)
+                    subdiv_data = create_subdiv_dict(subdiv_rows)
```

Comment:
Please move the subdiv call into a separate function to improve readability.
```suggestion
                    ........
                    subdiv_data = get_subdiv_data create_subdiv_dict(subdiv_rows)
                    ........
                    
                    
          def get_subdiv_rows(rows, cursor):
          """ comment the higlevel purpose of this function"""
               subdiv_rows = get_subdiv_rows(rows, cursor)
               subdiv_data = create_subdiv_dict(subdiv_rows)
               
               return subdiv_rows
          
```

Reply:
Done